### DISCUSSION TITLE
need IPV6 support for python version

### DISCUSSION THREAD

**Discussion by zzpop** [open-webui/open-webui#5824] [General]
**Is your feature request related to a problem? Please describe.**
IPv6 is not supported yet for python "pip install open-webui"

**Describe the solution you'd like**
Could someone please show me how to support IPV6 visit ? Maybe change Uvicorn configuration?

**Describe alternatives you've considered**
_

**Additional context**
_

**Comment by tjbck** [comment]
PR Welcome!

**Reply by upmcplanetracker** [reply]
I have one that should make this compltely dual stack -- ivp6 with robust ipv4 fallback, but will also work in ipv6 only environments.  will submit.

**Comment by gaby** [comment]
This is a problem I ran into on a different project. Uvicorn only supports binding to ipv4 or ipv6. You have to migrate to hypercorn to support both:

Example:

```console
hypercorn main:app --bind 0.0.0.0:3000 --bind [::]:3000
```
